config/env/production.js

```diff
@@ -47,13 +54,14 @@ if (AWSCreds) {
 
 // If running in Cloud Foundry with a redis service
 if (redisCreds) {
-  module.exports.session = {
-    adapter: 'redis',
+  session = _.extend({}, session, {
+    adapter: 'socket.io-redis',
```

Comment:
This was in the original code. I don't know why it's set to use that particular adapter, but it's not something I feel comfortable changing without being able to test the outcome.